frontend/packages/operator-lifecycle-manager/src/components/clusterserviceversion.tsx

```diff
@@ -422,8 +512,72 @@ export const ClusterServiceVersionList: React.SFC<ClusterServiceVersionListProps
   );
   const NoDataEmptyMsg = () => <MsgBox title="No Operators Found" detail={noDataDetail} />;
 
-  return (
+  // remove duplicate global operator entries
+  const filteredData =
+    data.length > 0
+      ? data.filter((source) => {
+          const kind = referenceFor(source);
+          if (source == null) {
+            return null;
+          }
+          if (referenceForModel(SubscriptionModel) === kind) {
+            return source;
+          }
+          if (referenceForModel(ClusterServiceVersionModel) === kind) {
+            if (source.metadata.annotations?.['olm.operatorGroup'] === 'global-operators') {
+              return source.metadata.namespace === 'openshift-operators' ? source : null;
```

Comment:
This logic is wrong for native k8s. The namespace could be something else. Ideally you'd find the right operator group and check its namespace, although I'm not sure that's possible for normal users.

We might have to circle back on this and decide what to do.

Reply:
@rebeccaalpert I would filter out any CSV with status "Copied." This is simpler and should work in all environments.

Reply:
Thanks, will do.